packages/embark-test-runner/src/test.js

```diff
@@ -260,11 +292,13 @@ class Test {
 
   async _deploy(config, callback) {
     const self = this;
+    let contractConfig = config.contracts || {};
     async.waterfall([
       function setConfig(next) {
-        prepareContractsConfig(config);
+        contractConfig = prepareContractsConfig(contractConfig);
         self.events.request('config:contractsConfig:set',
-          {contracts: config.contracts, versions: self.versions_default}, next);
+          // TODO find out what versions_default is where it went
```

Comment:
Has this been figured out by now?

Reply:
Nope, I didn't investigate much, but versions_default is used nowhere in the code base. 
I guess it's a relic of Embark's past. Juts gonna remove it

Reply:
Went in the time machine and I found it: https://github.com/embark-framework/embark/blob/3.2.x/lib/tests/test.js#L171
Apparently, it was used keep a copy of the versions so we don't reset anything. Considering we haven't had any issues in that regard since all that time, I think it's safe to say that the `versions_default` can go in retirement.

Reply:
Thanks for taking the time to verify this!